styles.css

```diff
@@ -372,6 +369,155 @@ section {
   line-height: 16px;
 }
 
+.popup-bg {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100%;
+  background-color: rgba(193, 199, 208, 0.9);
+}
+
+.popup {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  align-self: center;
+  height: 95vh;
+  width: 86%;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
```

Comment:
- [OPTIONAL] Both the see live and see source buttons do not show on the mobile popup. Kindly check them out and fix.

### Your Design
![popup-mobile](https://github.com/aamir-asaram/Portfolio/assets/26597432/de9b1503-31fd-4344-8e62-3cbd2e082410)

### Figma Design
![temp-popup-mobile](https://github.com/aamir-asaram/Portfolio/assets/26597432/20fd46a4-0fb1-4a11-bf2d-b2b418ec5830)